sdk/documenttranslation/Azure.AI.DocumentTranslation/src/DocumentTranslationClient.cs

```diff
@@ -175,14 +175,16 @@ public virtual DocumentTranslationOperation StartTranslation(Uri sourceBlobConta
                 Filter = options?.Filter
             };
 
-            var glossaries = glossary == null ? new List<TranslationGlossary>() : new List<TranslationGlossary> { glossary };
+            var target = new TranslationTarget(targetBlobContainerSas, targetLanguageCode)
+            {
+                Category = options?.Category
+            };
+
+            target.Glossaries.Add(glossary);
```

Comment:
because `glossary` is not required, should we check that it is not null before adding it? or it doesn't matter

Reply:
Yes, you're right. We should check.